### ISSUE TITLE
Is there a way to make a screenshot on a given selector only?

### ISSUE THREAD

**Issue by razvanvancea** [tacoss/testcafe-blink-diff#62]
I want to make a screenshot only on a section of the page. Is there a way to give a selector and then make always screenshots on it?

**Comment by pateketrueke** [comment]
Yeah, that's possible, the `takeSnapshot` method accepts a `selector` option, e.g.

```js
await takeSnapshot(t, { selector: string|Selector|Selector[] });
```

You can pass a string, or Selector instance, or an array of both, etc.
